nixos/modules/virtualisation/google-compute-image.nix

```diff
@@ -44,10 +44,24 @@ in
         GZIP compression level of the resulting disk image (1-9).
       '';
     };
+
+    virtualisation.googleComputeImage.efi = mkOption {
+      type = types.bool;
+      default = false;
+      description = lib.mdDoc ''
+        Whether to use EFI booting.
+      '';
+    };
```

Comment:
Can be replaced with `mkEnableOption` instead.

Reply:
Updated